### PR TITLE
IR-C100 && IR-R300

### PR DESCRIPTION
**Pull request by Sunspeed-Robotics**
Please post the robot information (IR-C100 && IR-R300)
Shanghai Shansu Intelligent Technology Co., Ltd.

### CODE REVIEW

File: _posts/2020-06-08-ir-c100.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- 'diff-drive'
+- 'elevator'
+- 'indoor'
+- 'logistics robot'
+- 'mobile robot'
+- 'research'
+- 'robotnik'
+twitter_text: Checkout the IR-C100
+website: http://amrobots.net/en/ir-c100-3/
+wiki_homepage: http://amrobots.net/en/ir-c100-3/
```
Comment (tfoote) [inline _posts/2020-06-08-ir-c100.md]: This is not a ROS wiki page with the ROS developer API documented.

File: _posts/2020-06-08-ir-c100.md~ [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (tfoote) [inline _posts/2020-06-08-ir-c100.md~]: This file appears to be accidentally committed.

File: _posts/2020-06-08-ir-r300.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- 'diff-drive'
+- 'elevator'
+- 'indoor'
+- 'logistics robot'
+- 'mobile robot'
+- 'research'
+- 'robotnik'
+twitter_text: Checkout the IR-R300
+website: http://amrobots.net/en/ir-m300-3/
+wiki_homepage: http://amrobots.net/en/ir-m300-3/
```
Comment (tfoote) [inline _posts/2020-06-08-ir-r300.md]: This also is not a wiki page with ROS developer documentation.

**Comment by Sunspeed-Robotics** [comment]
According to your request, the relevant information has been modified, please pass the review as soon as possible.

**Comment by tfoote** [comment]
The link to the source repository is good to have available. It's usually linked from the developer documentation as well as submitted for indexing. But that's not a wiki page on the ROS wiki with the developer documentation. I don't see any significant documentation.

From the contributing guide you should be adding the link to here with the documentation here. 
```
Developer documentation should remain on the ROS wiki, usually ‘http://wiki.ros.org/Robots/RobotName’. Where RobotName is substituted for your robot’s name.
```

**Comment by Sunspeed-Robotics** [comment]
According to your requirements, the wiki home page has been replaced. I hope you will pass the review as soon as possible.

**Comment by Sunspeed-Robotics** [comment]
According to your requirements, the wiki home page has been replaced.

**Comment by tfoote** [comment]
Thank you for creating the wiki pages. That's the first step. But as per our requirements you must also provide a publicly documented ROS interface which is what's typically supposed to go on that wiki page as developer documentation. I don't see any documentation of the ROS interface and browsing through the supporting links I didn't find any ROS references there either. You need to document the ROS interface to be listed as a ROS robot.

**Comment by Sunspeed-Robotics** [comment]
According to your request, I have compiled the relevant interfaces of the IR-C100 && IR-R300 robot. The URL is: http://wiki.ros.org/Robots/IR-C100　&& http://wiki.ros. org/Robots/IR-R300. Please review as soon as possible.

File: _posts/2020-06-08-ir-c100.md~ (hunk 2 above)
Comment (tfoote) [reply _posts/2020-06-08-ir-c100.md~]: This file still needs to be removed from the pull request.

File: _posts/2020-06-08-ir-r300.md~ [hunk 4]
```diff
@@ -0,0 +1,88 @@
+---
```
Comment (tfoote) [inline _posts/2020-06-08-ir-r300.md~]: This file should also be removed.

File: assets/css/main.css [hunk 5]
```diff
@@ -1 +1 @@
...
...-direction:row-reverse;-ms-flex-wrap:wrap;flex-wrap:wrap;max-width:800px;margin:auto;margin-top:3.125rem;top:2.063rem}.post-manipulator .post-category{width:100%}.post-manipulator .post-category a{color:#d7772e;border:1px solid #d7772e;border-radius:3px;padding:.313rem .625rem;margin:0 0 .313rem;z-index:50;display:inline-block}.post-manipulator .post-resources{width:100%}.post-manipulator .post-resources a{color:#d7772e;border:1px solid #d7772e;border-radius:3px;padding:.313rem .625rem;margin:0 0 .313rem;z-index:50;display:inline-block}.post-manipulator .post-tags{width:100%}.post-manipulator .post-tags a{color:#d7772e;border:1px solid #d7772e;border-radius:3px;padding:.313rem .625rem;margin:0 0 .313rem;z-index:50;display:inline-block}.post-manipulator aside{background-color:#d7772e}.post-manipulator aside nav li a:hover{color:#d7772e}.post-manipulator .ribbon span{background-color:#d7772e}.post-manipulator .ribbon span:before{border-left:3px solid #99531d;border-top:3px solid #99531d}.post-manipulator .ribbon span:after{border-right:3px solid #99531d;border-top:3px solid #99531d}.post-manipulator footer{background-color:#d7772e}.post-manipulator .question{border:1px solid #d7772e}
```
Comment (tfoote) [inline assets/css/main.css]: And this PR should not be changing the CSS and this should be reverted.

**Comment by Sunspeed-Robotics** [comment]
Delete the hidden files according to your requirements and replace the css files. I hope you can pass the review as soon as possible.

**Pull request merged**